bin/rad-diff

```diff
@@ -99,6 +99,14 @@
   (fn [commit]
     (string/unlines (process-with-stdout! "git" ["rev-parse" "--short" commit] ""))))
 
+(def get-commit-parent!
+  (fn [commit]
+    (string/unlines (process-with-stdout! "git" ["rev-parse" (string-append commit "^")] ""))))
+
+(def get-head-origin-master!
+  (fn []
+    (string/unlines (process-with-stdout! "git" ["rev-parse" "origin/master"] ""))))
```

Comment:
Maybe you could extract 

~~~clojure
(def git-rev-parse
  (fn [commitish] (string/unlines (process-with-stdout! "git" ["rev-parse" commitish] "")))
~~~

And then inline the calls to this function in `verify-patch`.

Reply:
I think this should be done in a different PR as there are more git rev parse commands (with options) and git commands in general could be more cleaned up.